AutomatedTesting/Gem/PythonTests/assetpipeline/scene_settings_tests/tests/scene_settings_tests_in_editor.py

```diff
@@ -0,0 +1,222 @@
+    scene_settings_file_update_disabled_after_save = (
+        "Scene settings UI update button has correctly become disabled after finishing the save operation.",
+        "Scene settings UI update button did not become disabled after finishing the save operation."
+    )
+
+    # Finally, make sure the scene settings file was actually created on disk.
+    scene_settings_file_created = (
+        "Scene settings file successfully created.",
+        "Failed to create scene settings file."
+    )
+    
+
+def is_this_the_update_materials_checkbox(check_box):
+    # The reflected property editor doesn't include names or unique identifiers. The labels are actually different objects.
+    # The easiest way to find the button is to go up a few parents to the ancestor with the tooltip, and check that.
+    # "Checking this box will accept changes made in the source file into the Open 3D Engine asset."
+    check_box_parent_1 = check_box.parent()
+    if not check_box_parent_1:
+        return None
+    check_box_parent_2 = check_box_parent_1.parent()
+    if not check_box_parent_2:
+        return None
+    check_box_parent_3 = check_box_parent_2.parent()
+    if not check_box_parent_3:
+        return None
+
+    if check_box_parent_3.toolTip() == "<b></b>Checking this box will accept changes made in the source file into the Open 3D Engine asset.":
+        return check_box_parent_3
+    return None
+
+
+def Scene_Settings_Tests_In_Editor_Create_And_Verify_Scene_Settings_File():
+    import pyside_utils
+
+    @pyside_utils.wrap_async
+    async def run_test():
+
+        import asyncio
+        from editor_python_test_tools.utils import Report
+        import PySide2
+        from PySide2 import QtWidgets
+        import azlmbr.bus as bus
+        import azlmbr.editor as editor
+        import azlmbr.legacy.general as general
+    
+        project_root_folder = editor.EditorToolsApplicationRequestBus(bus.Broadcast, 'GetGameFolder')
+        path_to_test_asset = os.path.join(project_root_folder, "auto_test_fbx.fbx")
+        path_to_manifest = path_to_test_asset + ".assetinfo"
+
+        # Make sure the test asset exists
+        Report.critical_result(Tests.scene_settings_test_asset_exists, os.path.exists(path_to_test_asset))
+
+        # Make sure the scene settings file doesn't yet exist
+        Report.critical_result(Tests.scene_settings_scene_settings_not_created_yet, not os.path.exists(path_to_manifest))
+
+        window_id = azlmbr.qt.SceneSettingsAssetImporterForPythonRequestBus(azlmbr.bus.Broadcast, "EditImportSettings", path_to_test_asset)
+
+        # This test needs to pause between some operations to let asynchronous operations finish.
+        general.idle_enable(True)
+
+        # The window doesn't immediately populate when opened, so wait a few frames for it to
+        # generate the interface using the reflected property system.
+        general.idle_wait_frames(30)
```

Comment:
Just curious: Is there any notification we can listen to instead of hardcoded the sleep time here?

Reply:
Not yet, I'll look into adding something when I write the next tests. I'll probably be re-writing this test after it goes in, anyways, so I'll revisit this soon.